.gitignore

```diff
@@ -34,4 +34,4 @@ dist
 es
 lib
 yarn-error.log*
-package-lock.json
+package-lock.json
```

Comment:
Does nuka have a lint precommit hook? This eliminated newline seems odd

Reply:
There was another line here in the original PR, but it wasn't necessary so I removed it. it was also part of a merge conflict so I'm assuming the reason it isn't showing up in the diff is because of that